Library/Homebrew/extend/ENV/std.rb

```diff
@@ -303,6 +303,10 @@ def libstdcxx
     end
   end
 
+  def java_cache
+    append "_JAVA_OPTIONS", "-Duser.home=#{HOMEBREW_CACHE}/java_cache"
+  end
```

Comment:
No worries! We all tend to read what we want to read instead of what is actually written. :smirk_cat: Just wanted to point out the difference, even if you've already moved the stuff.